cura/Layer.py

```diff
@@ -61,39 +68,59 @@ def createJumps(self):
         return self.createMeshOrJumps(False)
 
     def createMeshOrJumps(self, make_mesh):
-        builder = MeshBuilder()
-
+        builder = MeshBuilder() # This is never really used, only the mesh_data inside
+        mesh_data = builder._mesh_data
```

Comment:
This is changing with the simple_convex_hull branch, but this is actually a pretty bad practice. You are accessing a protected instance variable from outside. Note that python has no real concept of protected or private, but by convention anything starting with an underscore should at the very least be treated as protected. Note that there is a getter for MeshBuilder's mesh data (getMeshData) so there is no need for this. In addition, you actually would not need MeshBuilder and could just have used MeshData's constructor directly, but this is changing for the simple_convex_hull branch.